docs/source/en/model_doc/auto.mdx

```diff
@@ -78,222 +78,238 @@ Likewise, if your `NewModel` is a subclass of [`PreTrainedModel`], make sure its
+### AutoModelForSeq2SeqLM
 
-[[autodoc]] AutoModelForTableQuestionAnswering
+[[autodoc]] AutoModelForSeq2SeqLM
 
-## AutoModelForDocumentQuestionAnswering
+### TFAutoModelForSeq2SeqLM
 
-[[autodoc]] AutoModelForDocumentQuestionAnswering
+[[autodoc]] TFAutoModelForSeq2SeqLM
 
-## AutoModelForImageClassification
+### FlaxAutoModelForSeq2SeqLM
 
-[[autodoc]] AutoModelForImageClassification
+[[autodoc]] FlaxAutoModelForSeq2SeqLM
 
-## AutoModelForVideoClassification
+### AutoModelForSequenceClassification
 
-[[autodoc]] AutoModelForVideoClassification
+[[autodoc]] AutoModelForSequenceClassification
 
-## AutoModelForVision2Seq
+### TFAutoModelForSequenceClassification
 
-[[autodoc]] AutoModelForVision2Seq
+[[autodoc]] TFAutoModelForSequenceClassification
 
-## AutoModelForVisualQuestionAnswering
+### FlaxAutoModelForSequenceClassification
 
-[[autodoc]] AutoModelForVisualQuestionAnswering
+[[autodoc]] FlaxAutoModelForSequenceClassification
 
-## AutoModelForAudioClassification
+### AutoModelForMultipleChoice
 
-[[autodoc]] AutoModelForAudioClassification
+[[autodoc]] AutoModelForMultipleChoice
 
-## AutoModelForAudioFrameClassification
+### TFAutoModelForMultipleChoice
 
-[[autodoc]] AutoModelForAudioFrameClassification
+[[autodoc]] TFAutoModelForMultipleChoice
 
-## AutoModelForCTC
+### FlaxAutoModelForMultipleChoice
 
-[[autodoc]] AutoModelForCTC
+[[autodoc]] FlaxAutoModelForMultipleChoice
 
-## AutoModelForSpeechSeq2Seq
+### AutoModelForNextSentencePrediction
 
-[[autodoc]] AutoModelForSpeechSeq2Seq
+[[autodoc]] AutoModelForNextSentencePrediction
 
-## AutoModelForAudioXVector
+### TFAutoModelForNextSentencePrediction
 
-[[autodoc]] AutoModelForAudioXVector
+[[autodoc]] TFAutoModelForNextSentencePrediction
 
-## AutoModelForMaskedImageModeling
+### FlaxAutoModelForNextSentencePrediction
 
-[[autodoc]] AutoModelForMaskedImageModeling
+[[autodoc]] FlaxAutoModelForNextSentencePrediction
 
-## AutoModelForObjectDetection
+### AutoModelForTokenClassification
 
-[[autodoc]] AutoModelForObjectDetection
+[[autodoc]] AutoModelForTokenClassification
 
-## AutoModelForImageSegmentation
+### TFAutoModelForTokenClassification
 
-[[autodoc]] AutoModelForImageSegmentation
+[[autodoc]] TFAutoModelForTokenClassification
 
-## AutoModelForSemanticSegmentation
+### FlaxAutoModelForTokenClassification
 
-[[autodoc]] AutoModelForSemanticSegmentation
+[[autodoc]] FlaxAutoModelForTokenClassification
 
-## AutoModelForInstanceSegmentation
+### AutoModelForQuestionAnswering
 
-[[autodoc]] AutoModelForInstanceSegmentation
+[[autodoc]] AutoModelForQuestionAnswering
 
-## AutoModelForZeroShotObjectDetection
+### TFAutoModelForQuestionAnswering
 
-[[autodoc]] AutoModelForZeroShotObjectDetection
+[[autodoc]] TFAutoModelForQuestionAnswering
 
-## TFAutoModel
+### FlaxAutoModelForQuestionAnswering
 
-[[autodoc]] TFAutoModel
+[[autodoc]] FlaxAutoModelForQuestionAnswering
 
-## TFAutoModelForPreTraining
+### AutoModelForTableQuestionAnswering
```

Comment:
This one could be in multi-modal more than NLP. It does receive the table as text, but it doesn't accept generic texts.